python/desc/sims/GCRCatSimInterface/CatalogClasses.py

```diff
@@ -4,15 +4,91 @@
 from lsst.utils import getPackageDir
 from lsst.sims.catUtils.exampleCatalogDefinitions import PhoSimCatalogSersic2D
 from lsst.sims.catUtils.exampleCatalogDefinitions import PhoSimCatalogZPoint
+from lsst.sims.catUtils.exampleCatalogDefinitions import PhoSimCatalogSN
 from lsst.sims.catUtils.mixins import VariabilityAGN
 from lsst.sims.catalogs.decorators import cached, compound
 from lsst.sims.catUtils.mixins import EBVmixin
 
-__all__ = ["PhoSimDESCQA", "PhoSimDESCQA_AGN"]
+from lsst.sims.catalogs.db import fileDBObject
+
+__all__ = ["PhoSimDESCQA", "PhoSimDESCQA_AGN", "SNFileDBObject",
+           "DC2PhosimCatalogSN"]
 
 #########################################################################
 # define a class to write the PhoSim catalog; defining necessary defaults
 
+class SNFileDBObject(fileDBObject):
```

Comment:
The `SNFileDbObject` class should actually go in `DatabaseEmulator.py`.  That is where we are keeping the classes that behave like `CatalogDBObject`s.